components/LeagueEntries/LeagueEntries.test.tsx

```diff
@@ -50,4 +61,34 @@ describe('LeagueEntries', () => {
     );
     expect(leagueEntryNumber).toHaveTextContent('Entry 3');
   });
+
+  it('renders teamLogo when user makes a pick', () => {
```

Comment:
Why is there a comma after the `it` statement?

Reply:
@ryandotfurrer Could you please elaborate on this? As of the moment, I do not see a comma after the it statement?

Reply:
> @ryandotfurrer Could you please elaborate on this? As of the moment, I do not see a comma after the it statement?

@vmaineng Sure! `it('renders teamLogo when user makes a pick', () => {` Just after "... user makes a pick" there is a comma between that line of text and the ().

Reply:
@ryandotfurrer Ah, it is there to separate the description of the test case from the callback function that contains the actual test logic. As you can see in the previous tests in this file, there is a `,` after the string 'renders teamLogo.....'. Please let me know if this makes sense :)

Reply:
@vmaineng huh, I guess I've just not noticed it before. Thanks!